mailer.js

```diff
@@ -26,6 +27,28 @@ module.exports = function (log) {
     return 'href="' + url + '" style="color: #0095dd; text-decoration: none; font-family: sans-serif;"'
   }
 
+  function constructLocationString (location) {
+    // construct the location string from the location object
+    var locationString = ''
+    if (location !== null) {
+      if (location.city) {
+        locationString += location.city + ', '
+      }
+      locationString += location.country + ' ' + '(estimated)'
+    }
+    return locationString
+  }
+
+  function constructLocalTimeString (timeZone) {
+    // if no timeZone is passed, use 'America/Los_Angeles'
```

Comment:
I don't believe we discussed this scenario in meeting. @ryanfeeley I feel like the fallback should be UTC since that is the time displayed now without locale information. Thoughts?